DESCRIPTION

```diff
@@ -29,21 +29,21 @@ URL: https://github.com/insightsengineering/tern.mmrm,
 BugReports: https://github.com/insightsengineering/tern.mmrm/issues
 Depends:
     R (>= 3.6),
-    tern (>= 0.9.7)
+    tern (>= 0.9.9)
 Imports:
     checkmate (>= 2.1.0),
     cowplot,
     dplyr (>= 1.1.0),
     emmeans (>= 1.10.4),
-    formatters (>= 0.5.10),
+    formatters (>= 0.5.11),
     generics,
     ggplot2,
     lifecycle (>= 0.2.0),
     magrittr,
     mmrm (>= 0.3.5),
     parallelly (>= 1.25.0),
     rlang (>= 1.0.1),
-    rtables (>= 0.6.11),
+    rtables (>= 0.6.13),
```

Comment:
@shajoezhu should formatters/rtables/tern dependencies be updated to their latest versions? Tests were failing due to a change in {rtables} (that I corrected) but otherwise I don't think there's any downstream changes needed.

Reply:
thanks a lot for updating. I think we will also need to release this to cran then.